docs/_extra/api-reference/hypothesis-v1.yaml

```diff
@@ -102,7 +102,7 @@ info:
     | 500         | Server Error  | An error occurred with our API |
 
 servers:
-  - url: https://api.hypothes.is/api
+  - url: https://hypothes.is/api
```

Comment:
Thought I'd fix this while I was here: all the example URLs in our docs have been wrong, presumably since we first moved to redoc.